### PR TITLE
SEO Improvements 

### PR DESCRIPTION
**Pull request by 10thfloor**
Closes: #593 
Starts: #574

- [x] Not defined/correctly defined meta tags - meta tags on all docs pages are set to Start Building in the Open. Each page. 
- [x] Should have a better meta description. This one is important.
- [x] No language is defined in meta tags, Bing still uses those tags when crawling
- [x] There is no sitemap and no robots.txt

This was omitted. There is no accessibility story for the docs site.
- [ ] Alt texts are missing

### CODE REVIEW

File: docs/gatsby-config.js [hunk 1]
```diff
@@ -362,6 +364,28 @@ const sections = [
...
+        "[Types, Interfaces, and Definitions](/fcl/api#types-interfaces-and-definitions)"
+      ],
+      Changelogs: [
+        "[@onflow/fcl](/fcl/packages/fcl/CHANGELOG)",
+        "[@onflow/types](/fcl/packages/types/CHANGELOG)"
+      ],
+      "Developer Guides": [
+        "[Introducing @onflow/fcl](/fcl)",
+        "[Flow App Quickstart](/fcl/flow-app-quickstart)"
+      ]
```
Comment (10thfloor) [inline docs/gatsby-config.js:386]:
```suggestion
```

File: docs/plugins/gatsby-theme-flow/src/components/layout.js [hunk 2]
```diff
@@ -22,11 +22,7 @@ export default function Layout(props) {
         return (
           <Fragment>
             <Helmet defaultTitle={title} titleTemplate={`%s - ${title}`}>
-              <meta name="description" content={description} />
-              <link
-                rel="stylesheet"
-                href="https://use.typekit.net/ntx0wks.css"
-              ></link>
+              <html lang="en" />
```
Comment (10thfloor) [inline docs/plugins/gatsby-theme-flow/src/components/layout.js:25]:
```suggestion
             <link
                rel="stylesheet"
                href="https://use.typekit.net/ntx0wks.css"
              ></link>
              <html lang="en" />
```

Comment (10thfloor) [reply docs/plugins/gatsby-theme-flow/src/components/layout.js:25]: Might need this?

**Review by psiemens — APPROVED** [review]

**Review by sideninja — APPROVED** [review]
Awesome 🎉

**Pull request merged**